src/mtx/commands/uplink/resetPowerMaxDay.ts

```diff
@@ -0,0 +1,73 @@
+/**
+ * Uplink command to reset the maximum daily power.
+ *
```

Comment:
add "The corresponding downlink command: `resetPowerMaxDay`." as in other commands

Reply:
fixed